fsm_order_request_workers/models/fsm_person.py

```diff
@@ -0,0 +1,25 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, fields, api
+
+
+class FSMPerson(models.Model):
+    _inherit = "fsm.person"
+    
+    vehicle = fields.Boolean()
+    salary_currency_id = fields.Many2one('res.currency')
```

Comment:
It's supposed to be the same currency for every field, because `net_salary` will be calculated without any currency conversion (we'll directly add or subtract amounts, so they will be in the same currency).
So we should define only one currency for all the `Monetary` fields in this model.
Also I also miss the initialization for this field (please see https://github.com/odoo/odoo/blob/15.0/addons/purchase/models/purchase.py#L97)